### DISCUSSION TITLE
What license is associated with the docker directory?

### DISCUSSION THREAD

**Discussion by jrtitus** [syslog-ng/syslog-ng#4425] [Q&A] [answered]
The directory itself has no LICENSE file, there is no header in the Dockerfile, and the COPYING file does not specify which license applies.

**Comment by MrAnno** [comment, marked as answer]
The `docker` directory was migrated from the [syslog-ng-docker](https://github.com/balabit/syslog-ng-docker/blob/master/LICENSE) repo, which was originally licensed as `LGPL-2.1-or-later`.

Thanks for noticing the missing headers, I'll add them.
